doc/source/methods/modeldistillation.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[source](../api/alibi_detect.ad.model_distillation.rst)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model distillation adversarial and drift detector"
```

Comment:
To fit with the naming of the other components in the docs, please change to `Model Distillation`.